<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window_</class>
 <widget class="QMainWindow" name="main_window_">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>273</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="central_widget_">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>171</width>
      <height>279</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vertical_layout_">
     <property name="leftMargin">
      <number>4</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>4</number>
     </property>
     <item>
      <widget class="QLabel" name="status_">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="start_preview_btn_">
       <property name="text">
        <string>Start Preview</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="end_preview_btn_">
       <property name="text">
        <string>End Preview</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="take_picture_btn_">
       <property name="text">
        <string>Take Picture</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="start_video_recording_btn_">
       <property name="text">
        <string>Start Video</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stop_video_recording_btn_">
       <property name="text">
        <string>Stop Video</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save_last_20_seconds_btn_">
       <property name="text">
        <string>Save last 20 seconds</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="file_name_input_"/>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>273</width>
     <height>37</height>
    </rect>
   </property>
   <widget class="QMenu" name="menutest">
    <property name="title">
     <string>Options</string>
    </property>
   </widget>
   <addaction name="menutest"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
